app/Actions/Import/Exec.php

```diff
@@ -0,0 +1,273 @@
+		// then the subdirectories.  This way, if the process fails along the
+		// way, it's much easier for the user to figure out what was imported
+		// and what was not.
+		$this->setUpIgnoreList($path, $ignore_list);
+
+		$files = glob($path . '/*');
+		$filesTotal = count($files);
+		$filesCount = 0;
+		$dirs = [];
+		$lastStatus = microtime(true);
+
+		// Add '%' at end for CLI output
+		$percent_symbol = ($this->statusCLIFormatting) ? '%' : '';
+
+		$this->status_update('Status: ' . $origPath . ': 0' . $percent_symbol);
+		foreach ($files as $file) {
+			// Reset the execution timeout for every iteration.
+			set_time_limit(ini_get('max_execution_time'));
+
+			// Report if we might be running out of memory.
+			$this->memWarningCheck();
+
+			// Generate the status at most once a second, except for 0% and
+			// 100%, which are always generated.
+			$time = microtime(true);
+			if ($time - $lastStatus >= 1) {
+				$this->status_update('Status: ' . $origPath . ': ' . intval($filesCount / $filesTotal * 100) . $percent_symbol);
+				$lastStatus = $time;
+			}
+
+			// Let's check if we should ignore the file
+			if ($this->checkAgainstIgnoreList($file, $ignore_list)) {
+				$filesTotal--;
+				continue;
+			}
+
+			if (is_dir($file)) {
+				$dirs[] = $file;
+				$filesTotal--;
+				continue;
+			}
+
+			$filesCount++;
+			// It is possible to move a file because of directory permissions but
+			// the file may still be unreadable by the user
+			if (!is_readable($file)) {
+				$this->status_update('Problem: ' . $file . ': Could not read file');
+				Logs::error(__METHOD__, __LINE__, 'Could not read file or directory (' . $file . ')');
+				continue;
+			}
+			$extension = Helpers::getExtension($file, true);
+			$raw_formats = strtolower(Configs::get_value('raw_formats', ''));
+			$is_raw = in_array(strtolower($extension), explode('|', $raw_formats), true);
+			if (@exif_imagetype($file) !== false || in_array(strtolower($extension), $this->validExtensions, true) || $is_raw) {
+				// Photo or Video
+				if ($this->photo($file, $this->delete_imported, $albumID, $this->force_skip_duplicates, $this->resync_metadata) === false) {
+					$this->status_update('Problem: ' . $file . ': Could not import file');
+					Logs::error(__METHOD__, __LINE__, 'Could not import file (' . $file . ')');
+				}
+			} else {
+				$this->status_update('Problem: ' . $file . ': Unsupported file type');
+				Logs::error(__METHOD__, __LINE__, 'Unsupported file type (' . $file . ')');
+			}
+		}
+		$this->status_update('Status: ' . $origPath . ': 100' . $percent_symbol);
+
+		// Album creation
+		foreach ($dirs as $dir) {
+			// Folder
+			$album = null;
+			if ($this->force_skip_duplicates || Configs::get_value('skip_duplicates', '0') === '1') {
+				$album = Album::where('parent_id', '=', $albumID == 0 ? null : $albumID)
+					->where('title', '=', basename($dir))
+					->get()
+					->first();
+			}
+			if ($album === null) {
+				$create = resolve(AlbumCreate::class);
```

Comment:
This fails; `use` missing (and now that the use of `Photo\Create` is in another file, I guess it no longer needs to be renamed to `AlbumCreate`?).